Ginger/Ginger/WindowExplorer/Common/WindowExplorerPage.xaml.cs

```diff
@@ -811,19 +814,30 @@ private void RefreshWindowsButton_Click(object sender, RoutedEventArgs e)
         {
             UpdateWindowsList();
         }
-        
+
+        RecordingManager RecordingMngr = null;
```

Comment:
please remove all changes from WindowExplorerPage, you will connect the RecordingManager with UI on new page you will design

Reply:
Removed